### PR TITLE
Remove some unused functions and library

### PR DESCRIPTION
**Pull request by howardjohn**
[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[ ] Does not have any changes that may affect Istio users.

### CODE REVIEW

**Review by bianpengyuan — APPROVED** [review]

File: tests/integration/telemetry/util.go [hunk 1]
```diff
@@ -46,39 +39,3 @@ func PromDumpWithAttributes(cluster resource.Cluster, prometheus prometheus.Inst
 
 	return ""
 }
-
-func SendTraffic(ingress ingress.Instance, t *testing.T, msg, url, extraHeader string, calls int64) *fhttp.HTTPRunnerResults {
```
Comment (douglas-reid) [inline tests/integration/telemetry/util.go:50]: i didn't realize we had completely switched away from using this as a traffic generator. I wonder if this "utility" can go away, and be refactored under `tests/integration/telemetry/prometheus` or something as all that is left is the Dump methods.

**Review by douglas-reid — APPROVED** [review]

**Review by ericvn — APPROVED** [review]

**Review by nmittler — APPROVED** [review]

File: tests/integration/telemetry/util.go (hunk 1 above)
Comment (ldemailly) [reply tests/integration/telemetry/util.go:50]: Was looking into making sure I wouldn't break istio's usage of fortio as libray in upcoming update but see it is gone... is there some background as to why/what is the replacement for exact/programmable traffic generation in tests nowadays in istio's code base?

Comment (howardjohn) [reply tests/integration/telemetry/util.go:50]: pkg/test/echo

Comment (ldemailly) [reply tests/integration/telemetry/util.go:50]: yeah I found https://github.com/istio/istio/tree/master/pkg/test/framework/components/echo interesting!
thx

**Pull request merged**